libs/filagui/CMakeLists.txt

```diff
@@ -87,7 +87,9 @@ target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 # ==================================================================================================
 # Compiler flags
 # ==================================================================================================
-target_compile_options(${TARGET} PRIVATE
-        -Wno-deprecated-register
-        $<$<CONFIG:Release>:-ffast-math>
-)
+if(NOT MSVC)
```

Comment:
For future reference, our code style is `if (…)` not `if(…)`. You don't need to fix it in this PR.

Reply:
Yes, if you want to build exclusively with msvc on windows, that this could be cleaned up. I didn't know that was the plan, but it would probably make sense. Duly noted about the coding style - my preference too :-).

Reply:
@romainguy, it might be interesting to maintain the ability to build with clang on windows, for example if you wanted to compare runtime performance between the code generated by the two compilers.